### PR TITLE
ref #7 - Upgrade & release python3

### PR DESCRIPTION
**Pull request by aRkadeFR**
Ref #7 

## Upgrade to python3 & release

- [x] Move to pbr
- [x] Move license to GPLv3
- [x] Change meta data information
- [x] Change README for having a link to the wiki
- [x] Update CHANGELOG
- [x] Upgrade Version to 0.7
- [x] Tag release and test on pypi-test
- [x] Release on pypi

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```
Comment (wo0dyn) [inline LICENSE]: Whatever the license was before, I'm not really comfortable with this copyright update:

```diff
+Copyright (C) 2016  aRkadeFR
-Copyright (C) 2012 Rémy HUBSCHER <remy.hubscher@novapost.fr>
```

Comment (aRkadeFR) [inline LICENSE]: Updated to PeopleDoc ;)

Comment (zebuline) [inline LICENSE]: @Natim is still in AUTHORS: https://github.com/novafloss/check_po/blob/master/AUTHORS

Comment (aRkadeFR) [inline LICENSE]: plus the real author in package metadata: https://github.com/novafloss/check_po/blob/master/setup.cfg#L14
Which FMPOV is the reference.

Comment (ewjoachim) [inline LICENSE]: Don't we need the explicit agreement of all previous  (major) contributors to change a software license ?

Comment (aRkadeFR) [inline LICENSE]: I'm not very well aware of licensing and process around it. I only use GPLv3 license for my projects, as it is recommended by the fsf.
I'm free to amend it or change it again.

Comment (Natim) [inline LICENSE]: Yes please don't use the GPL for this small project.

File: AUTHORS [hunk 2]
```diff
@@ -1,5 +1,5 @@
-########################
-Authors and contributors
-########################
-
-* Rémy Hubscher <remy.hubscher@novapost.fr>
+Benoît Bryon <benoit@marmelune.net>
+Natim <hubscher.remy@gmail.com>
```
Comment (aRkadeFR) [inline AUTHORS]: This is generated with `pbr` from the git history

File: ChangeLog [hunk 3]
```diff
@@ -0,0 +1,56 @@
+CHANGES
+=======
+
+0.7
```
Comment (aRkadeFR) [inline ChangeLog]: same as before, automatic generation with pbr from git history

File: LICENSE (hunk 1 above)
Comment (aRkadeFR) [inline LICENSE]: @Natim I revert to What The Fuck License ?
(I would be interested by some explanation between these two licenses, as I mentioned, I only know the GPLv3).

**Pull request merged**